NineChronicles.Headless/GraphTypes/StandaloneSubscription.cs

```diff
@@ -166,6 +167,34 @@ public StandaloneSubscription(StandaloneContext standaloneContext)
                 Resolver = new FuncFieldResolver<MonsterCollectionStatus>(context => (context.Source as MonsterCollectionStatus)!),
                 Subscriber = new EventStreamResolver<MonsterCollectionStatus>(context => standaloneContext.MonsterCollectionStatusSubject.AsObservable()),
             });
+            AddField(new EventStreamFieldType
+            {
+                Name = $"{nameof(MonsterCollectionStatus)}ByAgent",
+                Arguments = new QueryArguments(
+                    new QueryArgument<NonNullGraphType<AddressType>>
+                    {
+                        Description = "A hex-encoded address of agent.",
+                        Name = "address",
+                    }
+                ),
+                Type = typeof(NonNullGraphType<MonsterCollectionStatusType>),
+                Resolver = new FuncFieldResolver<MonsterCollectionStatus>(context => (context.Source as MonsterCollectionStatus)!),
+                Subscriber = new EventStreamResolver<MonsterCollectionStatus>(SubscribeMonsterCollectionStatus),
+            });
+            AddField(new EventStreamFieldType
+            {
+                Name = $"{nameof(MonsterCollectionState)}ByAgent",
+                Arguments = new QueryArguments(
+                    new QueryArgument<NonNullGraphType<AddressType>>
+                    {
+                        Description = "A hex-encoded address of agent.",
+                        Name = "address",
+                    }
+                ),
+                Type = typeof(NonNullGraphType<MonsterCollectionStateType>),
+                Resolver = new FuncFieldResolver<MonsterCollectionState>(context => (context.Source as MonsterCollectionState)!),
+                Subscriber = new EventStreamResolver<MonsterCollectionState>(SubscribeMonsterCollectionState),
+            });
```

Comment:
This might not be related to this PR but what's the difference between `MonsterCollectionStatus` and `MonsterCollectionState`? The two words mean pretty much the same to me so I'm curious about their functions.